packages/attestation-service/package.json

```diff
@@ -31,6 +31,8 @@
     "@celo/utils": "^0.1.0",
     "bignumber.js": "^7.2.0",
     "body-parser": "1.19.0",
+    "bunyan": "1.8.12",
+    "bunyan-gke-stackdriver": "0.1.2",
```

Comment:
Doesn't this couple the service tightly with GCloud? Validators may want to host this on AWS/Azure

Reply:
`bunyan-gke-stackdriver` by @mcortesi is a plugin to `bunyan` to create an alternatively formatted output stream, it does not require any GCP specific functionality to build/run, and can be configured with `LOG_FORMAT=stackdriver`, so is opt-in